tests/test_resources/test_clusters/test_on_demand_cluster.py

```diff
@@ -201,6 +202,37 @@ def test_fn_to_docker_container(self, ondemand_aws_cluster):
     ####################################################################################################
     # Status tests
     ####################################################################################################
+
+    @pytest.mark.level("minimal")
```

Comment:
these tests should prob be in `test_cluster` no? doesn't really need to be limited to ondemand

Reply:
yeah, but we are using `cluster.teardown() ` which is an on-demand cluster method. Runhouse Cluster does not have such method. Moreover, seems like `runhouse stop` is not a part of runhouse cli API (not in the docs - is it on purpose?), so not sure if we should use it in the test to this use-case.

Reply:
why do we need to do teardown? i see those tests are being skipped anyways maybe we can remove them

Reply:
to test that once we call cluster.teardown() the updated status is sent to den. We could delete it for now since they are skipped, but I think we should add a separate test suite that runs tests related to cluster termination (I think we've previously talked about it). wdyt?

Reply:
you mean confirming that when you call teardown the status gets updated in Den? maybe i'm missing something but i don't see that in this test?